server/src/main/scala/net/wiringbits/validations/ValidateVerifiedUser.scala

```diff
@@ -1,12 +1,13 @@
 package net.wiringbits.validations
 
+import net.wiringbits.common.ErrorMessages.emailNotVerified
 import net.wiringbits.repositories.models.User
 
 object ValidateVerifiedUser {
   def apply(user: User): Unit = {
     if (user.verifiedOn.isDefined)
       ()
     else
-      throw new RuntimeException("The email is not verified, check your spam folder if you don't see the email.")
+      throw new RuntimeException(emailNotVerified)
```

Comment:
I'd suggest `ErrorMessages.emailNotVerified` instead because that makes it obvious that we are referencing an error message